resources/migrate-to-swift.md

```diff
@@ -47,10 +47,34 @@ If you are using Cocoapods to manage your Adobe Experience Platform Mobile SDK d
   pod 'AEPUserProfile'
 ```
 
-Save the `Podfile` and run `pod repo update` to update your local Cocoapods repository.
+Save the `Podfile` and run `pod repo update` to update your local CocoaPods repository.
 
 Once the previous command is complete, run `pod install` or `pod update` to update the application dependencies.
 
+### SPM
+
+You can now also use SPM to manage your Adobe Experience Platform Mobile SDK dependencies. To add the AEP Packages to your application, from the Xcode menu select:
+
+```
+File > Swift Packages > Add Package Dependency...
+```
+
+Enter the URL for the AEP package repositories: 
+
+- AEPCore package: `https://github.com/adobe/aepsdk-core-ios.git`
+- AEPUserProfile package: `https://github.com/adobe/aepsdk-userprofile-ios.git`
+
+When prompted, input a specific version or a range of versions, and choose all the `AEP*` libraries.
```

Comment:
Specify the Dependency Rule and the Project. Click `Add Package`
When prompted, select all AEP* libraries. Click `Add Package`

Reply:
Thinking if we should provide screenshots here 🤔

Reply:
Added text description, I think we should be good without screenshots.